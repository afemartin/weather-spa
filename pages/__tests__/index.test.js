/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../index.js'

describe('(Page) Index', () => {
  const props = {
    location: null,
    data: {},
    loading: false,
    error: null
  }
  it('renders correctly', () => {
    const page = renderer.create(<Index {...props} />)
    const tree = page.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
