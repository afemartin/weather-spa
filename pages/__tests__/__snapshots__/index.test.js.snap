// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Page) Index renders correctly 1`] = `
<div
  className="jsx-1725421838 container"
>
  <div
    className="jsx-1725421838 header"
  >
    <h1
      className="jsx-1725421838 title"
    >
      Weather SPA
    </h1>
    <div
      className="jsx-1725421838 unit"
    >
      <div
        className="ui buttons"
      >
        <button
          className="ui blue button"
          disabled={undefined}
          onClick={[Function]}
          role="button"
          tabIndex={undefined}
        >
          Fahrenheit
        </button>
        <div
          className="or"
          data-text={undefined}
        />
        <button
          className="ui button"
          disabled={undefined}
          onClick={[Function]}
          role="button"
          tabIndex={undefined}
        >
          Celsius
        </button>
      </div>
    </div>
  </div>
  <div
    className="jsx-1725421838 location"
  >
    <h2
      className="jsx-1725421838"
    >
      Location
    </h2>
    <form
      action={undefined}
      className="ui form"
      onSubmit={[Function]}
    >
      <div
        className="field"
      >
        <div
          className="ui icon input"
        >
          <input
            disabled={undefined}
            name="location"
            onChange={[Function]}
            placeholder="Eg. Tokyo"
            tabIndex={undefined}
            type="text"
          />
          <i
            aria-hidden="true"
            className="search icon"
          />
        </div>
      </div>
    </form>
  </div>
  <div
    className="jsx-1725421838 data"
  >
    <div
      className="ui grid"
    >
      <div
        className="row"
      >
        <div
          className="six wide computer sixteen wide mobile six wide tablet column"
        >
          <div
            className="jsx-1725421838 condition"
          >
            <h2
              className="jsx-1725421838"
            >
              Condition
            </h2>
          </div>
        </div>
        <div
          className="ten wide computer sixteen wide mobile ten wide tablet column"
        >
          <div
            className="jsx-1725421838 forecast"
          >
            <h2
              className="jsx-1725421838"
            >
              Forecast
            </h2>
            <div
              className="ui celled list"
              role="list"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="jsx-1725421838 footer"
  >
    <a
      className="jsx-1725421838"
      href="https://www.yahoo.com/?ilc=401"
      target="_blank"
    >
      <img
        className="jsx-1725421838"
        height="29"
        src="https://poweredby.yahoo.com/purple.png"
        width="134"
      />
    </a>
  </div>
</div>
`;
